### PR TITLE
[NUI] CanvasView: Add ClipPath and Mask feature for Drawable

### PR DESCRIPTION
**Pull request by JSUYA**
### Description of Change ###

Shape and DrawableGroup, which inherit drawable class,
can clipping and masking using ClipPath and Mask methods.

### API Changes ###

 - ACR:no

Dependency patch
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/261672/

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs [hunk 1]
```diff
@@ -84,6 +100,43 @@ public Vector4 BoundingBox
...
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// The pixels of mask drawable and own drawable are blended according to MaskType.
+        /// </summary>
+        /// <param name="mask">The mask drawable object.</param>
+        /// <param name="type">The masking type.</param>
+        /// <returns>True when it's successful. False otherwise.</returns>
```
Comment (myroot) [inline src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs:126]: For the consistency you do better to use exception to inform error instead returning bool

Comment (JSUYA) [reply src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs]: I modified code by changing the return type and throwing an exception.

File: src/Tizen.NUI/src/internal/Interop/Interop.Drawable.cs [hunk 2]
```diff
@@ -45,6 +45,14 @@ internal static partial class Drawable
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Drawable_GetBoundingBox")]
             public static extern global::System.IntPtr GetBoundingBox(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Drawable_SetClipPath")]
+            [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]
+            public static extern bool SetClipPath(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.Drawable.cs:51]: from next time, please use meaningful names for arguments. 
`jarg1` and `jarg2` are auto-generated names from the past SWIG tool.
meaningful name would help to get good readibility.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**